test/fail_compilation/fail19182.d

```diff
@@ -0,0 +1,18 @@
+// REQUIRED_ARGS: -c
+/*
+TEST_OUTPUT:
+---
+gigi
```

Comment:
What is this doing here?

Reply:
Since we are in semantic, the pragma(msg) is ran before checking for the next instruction.

Reply:
Ah, ok.